catalog/core/catalog-core-localstorageprovider/src/test/java/org/codice/ddf/catalog/content/impl/FileSystemStorageProviderTest.java

```diff
@@ -176,51 +163,112 @@ public InputStream openStream() throws IOException {
                 return IOUtils.toInputStream("Updated NITF");
             }
         };
-        ContentItem updateItem = new ContentItemImpl(id, byteSource, NITF_MIME_TYPE,
+        ContentItem updateItem = new ContentItemImpl(id,
+                byteSource,
+                NITF_MIME_TYPE,
                 mock(Metacard.class));
-        UpdateStorageRequest updateRequest = new UpdateStorageRequestImpl(
-                Collections.singletonList(updateItem), null);
+        UpdateStorageRequest updateRequest = new UpdateStorageRequestImpl(Collections.singletonList(
+                updateItem), null);
 
-        UpdateStorageResponse updateResponse = provider.update(updateRequest);
-        List<ContentItem> items = updateResponse.getUpdatedContentItems();
+        assertUpdateRequest(updateRequest);
+    }
+
+    @Test
+    public void testDelete() throws Exception {
+        CreateStorageResponse createResponse = assertContentItem(TEST_INPUT_CONTENTS,
+                NITF_MIME_TYPE,
+                TEST_INPUT_FILENAME);
+        String id = createResponse.getCreatedContentItems()
+                .get(0)
+                .getId();
+        DeleteStorageRequest deleteRequest =
+                new DeleteStorageRequestImpl(createResponse.getCreatedContentItems()
+                        .stream()
+                        .map(ContentItem::getMetacard)
+                        .collect(Collectors.toList()), null);
+        when(deleteRequest.getMetacards()
+                .get(0)
+                .getId()).thenReturn(id);
+
+        DeleteStorageResponse deleteResponse = provider.delete(deleteRequest);
+        List<ContentItem> items = deleteResponse.getDeletedContentItems();
         ContentItem item = items.get(0);
 
         LOGGER.debug("Item retrieved: {}", item);
         assertEquals(id, item.getId());
-        assertEquals(NITF_MIME_TYPE, item.getMimeTypeRawData());
+        assertThat(item.getFilename(), is(""));
```

Comment:
Could use the [`isEmptyString()`](http://hamcrest.org/JavaHamcrest/javadoc/1.3/org/hamcrest/Matchers.html#isEmptyString()) matcher.